rewatch/src/build.rs

```diff
@@ -522,14 +525,15 @@ fn log_deprecated_config_field(package_name: &str, field_name: &str, new_field_n
 // is watching this file.
 // we don't need to do this in an incremental build because there are no file
 // changes (deletes / additions)
-pub fn write_build_ninja(build_state: &BuildState) {
+pub fn write_build_ninja(build_state: &BuildCommandState) {
     for package in build_state.packages.values() {
         // write empty file:
         let mut f = File::create(package.get_build_path().join("build.ninja")).expect("Unable to write file");
         f.write_all(b"").expect("unable to write to ninja file");
     }
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
Trade-off, I tried to use a struct here instead but it lead to more cloning than I'd like so turning off the warning seemed more pragmatic.